Maple2.Server.Game/Manager/Config/BuffManager.cs

```diff
@@ -204,6 +351,20 @@ public class BuffManager : IUpdatable {
         if (Reflect?.SourceBuffId == id) {
             Reflect = null;
         }
+
+        foreach ((BasicAttribute attribute, float value) in buff.Metadata.Status.Resistances) {
+            Resistances[attribute] = Math.Min(0, Resistances[attribute] - value);
+        }
+
+        Invokes.RemoveAll(id);
+        Compulsions.RemoveAll(id);
+
+        if (buff.Metadata.Status.Values.Any() || buff.Metadata.Status.Rates.Any() || buff.Metadata.Status.SpecialValues.Any() || buff.Metadata.Status.SpecialRates.Any()) {
```

Comment:
We should use `.Count > 0` here. It's a lot faster.

```cs
if (buff.Metadata.Status.Values.Count > 0 || buff.Metadata.Status.Rates.Count > 0 || buff.Metadata.Status.SpecialValues.Count > 0 || buff.Metadata.Status.SpecialRates.Count > 0) {
```

https://www.tabsoverspaces.com/233649-comparing-speed-of-count-0-and-any